src/screens/Filter/components/FilterItem/FilterItem.tsx

```diff
@@ -1,11 +1,31 @@
 import React from 'react';
-import { View } from 'react-native';
+import { View, Text } from 'react-native';
+
+import { PickListItem } from '@UI/PickListItem';
+import { FilterLabelsMapModel, FilterLabelsMap, FilterValuesMap, FilterValuesMapModel } from '@UI/FilterMap';
 
 import { styles } from './styles';
 
-export function FilterItem(): JSX.Element {
+import { FilterItemValueModel } from 'src/models';
+
+interface Props {
+  filterName: string;
+  values: Array<FilterItemValueModel>;
+}
 
+export function FilterItem({ filterName, values }: Props): JSX.Element {
   return (
-    <View style={styles.filterScreenContainer} />
+    <View style={styles.filterScreenContainer}>
+      <Text style={styles.filterLabel}>{FilterLabelsMap[filterName as keyof FilterLabelsMapModel]} :</Text>
```

Comment:
filterName should be typed differently(not as string), in order to avoid such statements.
In general - try to always avoid "as" casting